.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**What steps did you take and what happened:**
+[A clear and concise description of what the bug is.]
+
+**What did you expect to happen:**
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+**Environment:**
+
+- Heptio product version: 
```

Comment:
I'd prefer this to be repo-specific, if possible: `Sonobuoy version` in this case.

Reply:
Sounds good.